pmlc/conversion/stdx_to_llvm/tests/reshape.mlir

```diff
@@ -9,11 +9,11 @@ module {
 
 // CHECK-LABEL: func @reshaper0
 // CHECK: undef : !llvm<"{ float*, float*, i64, [3 x i64], [3 x i64] }">
-// CHECK: [[index0:%[0-9]+]] = llvm.mlir.constant(60 : index)
-// CHECK: insertvalue [[index0]], %{{[0-9]+}}[3, 0]
-// CHECK: [[index1:%[0-9]+]] = llvm.mlir.constant(1 : index)
-// CHECK: insertvalue [[index1]], %{{[0-9]+}}[4, 0]
-// CHECK: return {{%[0-9]+}} : !llvm<"{ float*, float*, i64, [1 x i64], [1 x i64] }">
+// CHECK: [[index0:%.+]] = llvm.mlir.constant(60 : index)
```

Comment:
But you can put `%` outside the regex, because MLIR vars will always start with `%`